engine/admin/logging/journald.md

```diff
@@ -87,4 +87,13 @@ logs:
     reader.add_match('CONTAINER_NAME=web')
 
     for msg in reader:
-      print '{CONTAINER_ID_FULL}: {MESSAGE}'.format(**msg)
\ No newline at end of file
+      print '{CONTAINER_ID_FULL}: {MESSAGE}'.format(**msg)
+
+## Journald configuration
+
+Docker hosts with many containers may produce massive amounts of log-messages.
+The default configuration of journald is to limit the number of messages stored per service per time-unit.
+
+Applications with need for large-scale logging wil probably need to configure `RateLimitIntervalSec` and `RateLimitBurst` in the journald configuration file.
```

Comment:
typo: "wil"

Consider rewording because applications don't configure. "If your applications need large-scale logging, configure `RateLimitIntervalSec` and `RateLimitBurst` in the `journald` configuration file."